index.html

```diff
@@ -5,11 +5,42 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="index.css">
     <title>gabriel's portfolio</title>
 </head>
 
 <body>
+    <div class="container">
+        <header>
+            <div class="single-shape">
+                <div class="shapes triangle">
 
+                </div>
+                <div class="shapes square">
+
+                </div>
```

Comment:
- [ ] [OPTIONAL] Its best to remove the unnecessary blank space here and in other places.